lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -556,12 +555,15 @@ def max_thread_limit
   system("rake -T")
 end
 
-RuboCop::RakeTask.new(:rubocop) do |task|
-  # These make the rubocop experience maybe slightly less terrible
-  task.options = ['-D', '-S', '-E']
-end
-
-RuboCop::RakeTask.new("rubocop:auto_correct") do |task|
```

Comment:
yeah. rubocop rake tasks automatically add a :auto_correct task using the base task's options.